src/routing_node.rs

```diff
@@ -74,15 +73,27 @@ impl RoutingNode {
                -> RoutingNode {
 
         let (crust_sender, crust_receiver) = ::std::sync::mpsc::channel::<::crust::Event>();
-        let cm = ::crust::ConnectionManager::new(crust_sender);
-        let accepting_on = cm.get_own_endpoints();
+        let mut crust_service = match ::crust::Service::new(crust_sender) {
+            Ok(service) => service,
+            Err(what) => panic!(format!("Unable to start crust::Service {}", what)),
+        };
+
+        let accepting_on = crust_service.start_default_acceptors().into_iter()
```

Comment:
No, the `Service::start_default_acceptors()` returns simply a `Vec<io::Result<Endpoint>>` which are the (local) endpoints crust is now accepting on.